akka-docs/src/main/paradox/futures.md

```diff
@@ -1,5 +1,15 @@
 # Futures
 
+## Dependency
+
+To use Future with Akka actors, you must add the following dependency in your project.
```

Comment:
This section explains using plain Scala Futures but focuses on their interop with Akka Actors, so to follow those examples you will want to depend on...